modules/loader.js

```diff
@@ -559,6 +567,131 @@ __private.loadBlockChain = function() {
 		});
 };
 
+/**
+ * Validate given block
+ *
+ * @param {object} block
+ * @param {function} cb
+ * @returns {setImmediateCallback} cb, err
+ */
+__private.validateBlock = (blockToVerify, cb) => {
+	const lastBlock = modules.blocks.lastBlock.get();
+
+	modules.blocks.utils.loadBlockByHeight(
+		blockToVerify.height - 1,
+		(lastBlockError, secondLastBlockToVerify) => {
```

Comment:
`lastBlockError` and `secondLastBlockToVerify`? Shouldn't it be `secondLastBlockError`?